src/Stratis.Bitcoin.Features.MemoryPool/MempoolValidator.cs

```diff
@@ -765,7 +765,7 @@ private void CheckSigOps(MempoolValidationContext context)
             // itself can contain sigops MAX_STANDARD_TX_SIGOPS is less than
             // MAX_BLOCK_SIGOPS; we still consider this an invalid rather than
             // merely non-standard transaction.
-            if (context.SigOpsCost > this.ConsensusOptions.MaxBlockSigopsCost)
+            if (context.SigOpsCost > this.ConsensusOptions.MaxStandardTxSigopsCost)
```

Comment:
@zeptin found this bug so lets let him have a look as well, I think also @majikandy had issues here.

Reply:
I'm pretty sure this `if` is now correct, at least as far as stratisX is concerned. I agree that Andy's issue is probably not resolved by this PR and still needs to be looked at.